src/coreComponents/physicsSolvers/fluidFlow/wells/WellSolverBase.hpp

```diff
@@ -303,6 +303,9 @@ class WellSolverBase : public SolverBase
   /// the number of Degrees of Freedom per reservoir element
   localIndex m_numDofPerResElement;
 
+  /// copy of the time step size saved in this class for residual normalization
+  real64 m_currentDt;
```

Comment:
Not a task for this PR, but perhaps we should be passing `dt` into `CalculateResidualNorm()`

Reply:
Yes I agree that it would be useful to have the `dt` passed as an argument of `calculateResidualNorm` here and maybe for others as well. I can take care of this in another PR.